config/clock.rb

```diff
@@ -10,9 +10,9 @@ class Clock
 
   # More-than-hourly jobs
 
-  every(10.minutes, 'IncrementalSyncAllFromTeacherTrainingPublicAPI', skip_first_run: true) do
-    TeacherTrainingPublicAPI::SyncAllProvidersAndCoursesWorker.perform_async(true)
-  end
+  # every(10.minutes, 'IncrementalSyncAllFromTeacherTrainingPublicAPI', skip_first_run: true) do
```

Comment:
We'll need to silence `CourseSyncNotSucceededForAnHour
CourseSyncNotSucceededForAnHour` when this is merged, hopefully not for long, and then run the full sync after to catchup

Reply:
👍